README.md

```diff
@@ -214,9 +215,22 @@ Example:
 
 # Contributors and Instructors
 
+## Create a dashboard (CSV file) about  exercise submissions
+
```

Comment:
The GitHiub have API call limits when not using any authentication, so, it would be good to add that the script checks if there is both GH_USER and GH_TOKEN env configured to do an authenticated api request, otherwise it will do it anonymously, which may be limited and will result in a wrong result.

Would be good to add those two ENVs that can be configured.